lib/pento_web/live/admin/age_filter.ex

```diff
@@ -0,0 +1,24 @@
+defmodule PentoWeb.Admin.AgeFilter do
+  import PentoWeb.Admin.Helpers
+
+  @age_group_filter %{
+    "all" => %{order: 1},
+    "18 and under" => %{order: 2, min: year(18)},
+    "18 to 25" => %{order: 3, min: year(25), max: year(15)},
+    "25 to 35" => %{order: 4, min: year(35), max: year(25)},
+    "35 and up" => %{order: 5, max: year(35)}
+  }
+
+  def options do
+    Map.to_list(@age_group_filter)
```

Comment:
Maybe AgeFilter is a Component, and the rendering logic can live with options.

Reply:
Could be, this will give yet another argument for `Age filter` creation